### ISSUE TITLE
Horizontal bar plot doesn't work with theme

### ISSUE THREAD

**Issue by vopani** [h2oai/wave#801]
#### Wave SDK Version, OS
v0.15.0 in Chrome on Mac

#### Actual behaviour
I am unable to create a horizontal bar plot if I set a theme in `meta_card`.

App 1:
```python
from synth import FakeCategoricalSeries
from h2o_wave import main, Q, app, copy_expando, data, ui

@app('/demo')
async def serve(q: Q):
    n = 20
    f = FakeCategoricalSeries()
    v = q.page.add('example', ui.form_card(
        box='1 1 4 5',
        items=[
            ui.visualization(
                plot=ui.plot([ui.mark(type='interval', x='=price', y='=product')]),
                data=data(fields='product price', rows=[(c, x) for c, x, _ in [f.next() for _ in range(n)]], pack=True),
            )
        ],
    ))

    await q.page.save()
```

Output 1:
<img width="579" alt="Screenshot 2021-05-08 at 10 35 35 PM" src="https://user-images.githubusercontent.com/8954217/117547620-246d1b80-b04e-11eb-8c8c-e653134edc88.png">

App 2:
```python
from synth import FakeCategoricalSeries
from h2o_wave import main, Q, app, copy_expando, data, ui

@app('/demo')
async def serve(q: Q):
    q.page['meta'] = ui.meta_card(box='', theme='light')

    n = 20
    f = FakeCategoricalSeries()
    v = q.page.add('example', ui.form_card(
        box='1 1 4 5',
        items=[
            ui.visualization(
                plot=ui.plot([ui.mark(type='interval', x='=price', y='=product')]),
                data=data(fields='product price', rows=[(c, x) for c, x, _ in [f.next() for _ in range(n)]], pack=True),
            )
        ],
    ))

    await q.page.save()
```

Output 2:
<img width="573" alt="Screenshot 2021-05-08 at 10 35 51 PM" src="https://user-images.githubusercontent.com/8954217/117547634-35b62800-b04e-11eb-8884-507aaa6d5669.png">

The only difference between App 1 and App 2 is `q.page['meta'] = ui.meta_card(box='', theme='light')` and I don't expect Output 2 to change from Output 1.

#### Expected behaviour
Should display horizontal bar plot in both examples (App 2 as well).

#### Steps To Reproduce
Run the two app codes.

#### Comments
I don't know if this is related to theme or something else altogether. But it definitely doesn't work while setting theme.

I also don't know if this is specific to bar plots or others too. I haven't checked.

**Comment by mturoci** [comment]
Interesting. After a bit of playing I figured that `fields` in `data` need to be reversed:
```py
from .synth import FakeCategoricalSeries
from h2o_wave import main, Q, app, copy_expando, data, ui

@app('/demo')
async def serve(q: Q):
    q.page['meta'] = ui.meta_card(box='', theme='light')

    n = 20
    f = FakeCategoricalSeries()
    v = q.page.add('example', ui.form_card(
        box='1 1 4 5',
        items=[
            ui.visualization(
                plot=ui.plot([ui.mark(type='interval', x='=price', y='=product')]),
                data=data(fields='price product ', rows=[(c, x) for c, x, _ in [f.next() for _ in range(n)]], pack=True),
            )
        ],
    ))

    await q.page.save()
```
Not sure about the connection to `theme` though yet.

Thanks for reporting @vopani!

**Comment by mturoci** [comment]
Here is transposed chart working regardless of the theme:
```py
from .synth import FakeCategoricalSeries
from h2o_wave import main, Q, app, copy_expando, data, ui

@app('/demo')
async def serve(q: Q):
    q.page['meta'] = ui.meta_card(box='', theme='light')

    n = 20
    f = FakeCategoricalSeries()
    v = q.page.add('example', ui.plot_card(
        box='1 1 4 5',
        title='Interval',
        data=data('product price', n),
        plot=ui.plot([ui.mark(type='interval', x='=price', y='=product', y_min=0)])
    ))
    v.data = [(c, x) for c, x, dx in [f.next() for _ in range(n)]]

    await q.page.save()
```

**Comment by vopani** [comment]
@mturoci I face the issue in `ui.visualization` in form_card only, and not with `ui.plot_card`. The plot_card works fine.

**Comment by mturoci** [comment]
Resolved by #746.
